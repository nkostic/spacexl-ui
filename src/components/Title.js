import React from "react";

class Title extends React.Component {
  render() {
    return (
      <section className="hero">
        <div className="hero-body">
          <div className="container">
            <h1 className="title has-text-centered has-text-white">
              {this.props.children}
            </h1>
          </div>
        </div>
      </section>
    );
  }
}

export default Title;
