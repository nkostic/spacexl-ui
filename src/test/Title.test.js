import React from 'react';
import { render } from '@testing-library/react';
import Title from '../components/Title';

test('<Title></Title>', () => {
  const {getByText, container} = render(<Title>SpaceX Launches</Title>);
  const titleElement = getByText(/SpaceX Launches/i);
  expect(titleElement).toBeInTheDocument();
  expect(container.firstChild).toMatchSnapshot();
})
