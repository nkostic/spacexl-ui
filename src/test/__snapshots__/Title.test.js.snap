// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title></Title> 1`] = `
<section
  class="hero"
>
  <div
    class="hero-body"
  >
    <div
      class="container"
    >
      <h1
        class="title has-text-centered has-text-white"
      >
        SpaceX Launches
      </h1>
    </div>
  </div>
</section>
`;
